### PR TITLE
GCW-3347 fixed item action icons on history page

### PR DESCRIPTION
**Pull request by swatijadhav**
### Ticket Link: 
https://jira.crossroads.org.hk/browse/GCW-3347

### What does this PR do?

fixed item action icons on history page

### CODE REVIEW

**Review by shreyas098 — APPROVED** [review]

File: app/constants/action-icons.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+export const ACTIONS_ICONS = [
```
Comment (patrixr) [inline app/constants/action-icons.js:1]: This seems to be a duplicate of `constants/item-actions`

Also, `Edited`, `Inventory` and `Pack` shouldn't be in the bottom menu.

`Edited` isn't a valid inventory action at all

Comment (patrixr) [reply app/constants/action-icons.js:1]: Can you point out what the actual change is ?

Comment (patrixr) [reply app/constants/action-icons.js:1]: Also note that the title of the PR is "fix icons on history page", but it's modifying a helper that can be used _everywhere_

Comment (swatijadhav) [reply app/constants/action-icons.js:1]: `constants/item-actions` this is specifically used in the actions dropdown next to designate button on item-published tab. And those are specific to item gain-loss.

On history tab, the actions like `pack`, `Edited`(groups item attribute updates), `inventory` are listed which are missing in  `constants/item-actions` and hence added different files only specific for icons.

Comment (patrixr) [reply app/constants/action-icons.js:1]: Aaah, gotcha ! I thought these were for the bottom menu. Looks good then :)

**Review by patrixr — APPROVED** [review]

**Pull request merged**